clr-k8s-examples/10-cri-resource-manager/README.md

```diff
@@ -0,0 +1,58 @@
+CRI Resource Manager
+====================
+CRI Resource Manager serves as a relay/proxy between kubelet and the container runtime,
+relaying requests and responses back and forth between these two, potentially alrering requests
+as they fly by.
+
+This document explains a very simple use case for the `cri-resource-manager`, for more details and tweaks
+on CRI Resource Manager service, you can go to https://github.com/intel/cri-resource-manager.
+
+Install
+-------
+[`install.sh`](install.sh) script will download the binary and install it as an `systemd` service unit. Below you can see the available
+variables you can use to customize the usage of your CRI Resource Manager service.
```

Comment:
nit: probably didn't mean to have line break here

Reply:
Thanks @jascott1, fixed.